From 8ff4e555dc63062a6395beea6dd8b09d9c197a98 Mon Sep 17 00:00:00 2001
From: Sam Mortimer <sam@mortimer.me.uk>
Date: Fri, 1 Dec 2017 09:07:02 -0800
Subject: [PATCH 01/19] Revert "sepolicy: Allow platform app to find nfc
 service"

This reverts commit a69f0be827871afbef32a8fdf95682e4b257ea9f.

We have this in device/lineage/sepolicy since it's not qcom specific.

Change-Id: Id20bc2ba3b9746f10fbf1671eccb0742ea1005c8
---
 vendor/common/platform_app.te | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/vendor/common/platform_app.te b/vendor/common/platform_app.te
index a8995aac..a7e04d20 100644
--- a/vendor/common/platform_app.te
+++ b/vendor/common/platform_app.te
@@ -10,9 +10,6 @@ binder_call(platform_app, secotad)
 
 allow platform_app imsrcs_service:service_manager find;
 
-# Allow NFC service to be found
-allow platform_app nfc_service:service_manager find;
-
 #Allow platform apps to interact with seemp health daemon
 binder_call(platform_app, seemp_health_daemon)
 
-- 
2.17.1

